src/shared/qr-code.ts

```diff
@@ -2,7 +2,15 @@ import { getReferenceType } from "../shared/utils.js";
 import { Data } from "./types";
 import { qrcodegen } from "./qr-code-generator.js";
 
-export default function generateQRCode(data: Data, size: number): string {
+function number(n: number) {
```

Comment:
What does this function do? Can we improve the naming, so that its intention is more clear?

Reply:
This function comes directly from [ PDFKit  PDFObject](https://github.com/foliojs/pdfkit/blob/120c3f9519e49d719a88d22d70139cc9f54d17d8/lib/object.js), the problem is that the class PDFObject is not exported... From my understanding, it is needed since the PDF standard has architectural limits [(PDF Reference Appendix C: Implementation Limits)](https://opensource.adobe.com/dc-acrobat-sdk-docs/pdfstandards/pdfreference1.7old.pdf)

Reply:
So it rounds numbers to the 6th decimal place and also limits the maximum and minimum. I would probably still rename it to something like `limitNumberAccordingToPDFSpecs` or `limitNumber` and put a documentation comment above it.

```suggestion
/**
 * Limits the maximum and minimum number possible according to the PDF specifications.
 * Borrowed from: https://github.com/foliojs/pdfkit/blob/120c3f9519e49d719a88d22d70139cc9f54d17d8/lib/object.js#L123-L130
 */
function limitNumber(n: number) {
```